app/src/main/res/values/strings.xml

```diff
@@ -139,6 +139,12 @@
     <string name="apply_text">APPLY</string>
     <string name="unaply_text">UNAPPLY</string>
     <string name="failed">failed</string>
+    <string name="enter_comment">Enter Comment</string>
+    <string name="send_comment_icon">send_comment_icon</string>
+    <string name="march_11">March 11</string>
+    <string name="john_doe">John Doe</string>
+    <string name="hello_how_are_you">Hello! How are you?</string>
+    <string name="_1_00">1:00</string>
```

Comment:
It seems a little strange to have some random default message/date/name for comments.

Reply:
I agree but in any case this will be deleted when we connect it to firebase so we can keep them